tests/test.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once 'src/php.php';
+
+class createpigmaDIVStest extends TestCase
+{
+    public function testcreatepigmaDIVSWrongdata(): void
+    {
+        $mockData =
+            [
+                ['id' => null]
+            ];
+
+
+        $expectedOutput = '';
+
+        $actualOutput = createpigmaDIVS($mockData);
+        $this->assertEquals($expectedOutput, $actualOutput);
+    }
+
+    public function testcreatepigmaDIVSrightData(): void
```

Comment:
What kind of test is this? Is it a success, a failure, or a malformed test?

It would be standard to name the test function like 
```suggestion
    public function testcreatepigmaDIVS_success(): void
```

So that the kind of test is clearly included